docs/medical-api/api-reference/patient/get-patient-consolidated.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Get Patient's Consolidated Data"
+description: "Gets a Patient's Consolidated Data."
+api: "GET /medical/v1/patient/{id}/consolidated"
+---
+
```

Comment:
Product wise - would it be a good idea to explain how this API fits into the full MAPI flow, and include a reference this endpoint in the Quickstart guide?